src/components/checkboxFilter/index.tsx

```diff
@@ -18,35 +18,64 @@ function CheckboxFilter<ItemKey extends string, FilterName extends string>({
   const hasGroups = items.some(
     (item) => (item.childCheckboxes ?? []).length > 0,
   );
+
+  const itemsPerColumn = Math.ceil(items.length / numberOfColumnsOnDesktop);
+
+  // Create an array to store items grouped by columns
```

Comment:
maybe to create a function `groupItemsByColumns` would make the comment unnecessary?